libnavigation-base/src/test/java/com/mapbox/navigation/base/internal/factory/RoadFactoryTest.kt

```diff
@@ -0,0 +1,64 @@
+package com.mapbox.navigation.base.internal.factory
+
+import com.mapbox.navigation.base.ExperimentalMapboxNavigationAPI
+import com.mapbox.navigator.NavigationStatus
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Assert
+import org.junit.Test
+
+@OptIn(ExperimentalMapboxNavigationAPI::class)
+class RoadFactoryTest {
+
+    @Test
+    fun `road object provides name`() {
+        val status = createNavigationStatus()
+
+        val road = RoadFactory.buildRoadObject(status)
+
+        Assert.assertEquals("roadName1", road.name)
```

Comment:
NIT

```suggestion
        assertEquals("roadName1", road.name)
```

Same thing applies to the rest of the `Assert`s below 👇

Reply:
Opting not to fix since that would require another CI run and block the release further.